patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -8,6 +8,15 @@
          }
      }
  
+@@ -265,7 +266,7 @@
+         this.func_78477_e();
+         this.func_78470_f();
+         this.field_78535_ad = this.field_78539_ae;
+-        this.field_78491_C = 4.0F;
++        this.field_78491_C = field_78490_B;
```

Comment:
Please add an "// FORGE:" style comment documenting why the patch is made. Will help future maintainers.

Reply:
You're' right, will do!